design/decisions/why-standardisation/index.qmd

```diff
@@ -0,0 +1,71 @@
+
+> How do we ensure that we work most efficiently, and have all the information we need to do our job to hand, when we need it?
+
+Not everything can be solved by standardisation, but quite a few workflow will always benefit from having a standard list of items that must be present before the next step in the process is initialised.
+
+## Decision Drivers
+
+As we have expanded the team it has become increasingly clear that we need some more standardised ways of dealing with some of the issues that a team of people encounter when working on a shared project.  There are for instance a number of things that we need to have present when reviewing pull request, or merging branches.  A number of these challenges can be overcome by setting standards for how we present data, and how we hand tasks over internally within the team.
+
+## Considered Options
+
+### Templates
+
+Templates helps with streamlining workflows, where implemented they enforce certain requirements regarding the information needed for a workflow to commence, and they help with ensuring that all the data needed to make a decision is present.  Templates can be used in many areas, we have (for now) focused on implementing them in the GitHub workflows. We are focusing on setting up templates for creating issues, and at the other end submitting pull requests.  In between we are recommending an extension for commits (Conventional Commits) which guides you through submitting a commit with more sensible comments.  
+
+### Code snippets in VS Code 
+
+User generated code snippets is a tool that was made available in Visual Studio Code back in 2018.  There is similar functionality in other editors, but VSCode seems to be the one that have support for markdown and quarto.  Code snippets rely on tab completion being enabled, but once that has been set it is very easy to create a set of custom snippets.  For the seedcase team we have (for now) focused on creating snippets that will allow us to use the same markdown/quarto formatting when writing our documentation.  There are snippets for creating tables as well as standardising comments, more information on how to use code snippets can be found here.
+
+<!--TODO link to doc on code snippets once we decide where to place it -->
+
+<!--TODO Check with Richard if pyCharm has similar function for 'text' editing-->
+
+### Joint list of key words for documentation
+
+Quarto allows for the generation of key words that can be used to filter documentation.  It is important for usability of the website that the words used are the same, and spelled the same, so that if a person is looking for how Seedcase handles databases they get everything by clicking on the keyword database, and aren't missing half the documents because they are tagged with the key word databases.  
```

Comment:
add quotation marks around "database" and "databases" when referring to a keyword